python-packages/smithy-http/smithy_http/__init__.py

```diff
@@ -178,6 +178,9 @@ def __len__(self) -> int:
     def __repr__(self) -> str:
         return f"Fields({self.entries})"
 
+    def __contains__(self, key: str) -> bool:
+        return key in self.entries
```

Comment:
Shouldn't we be normalizing the key (using `_normalize_field_name`) before checking if it exists `entries`? Similar to how we do it for the set, get, and remove methods?